setup.cfg

```diff
@@ -1,2 +1,4 @@
 [metadata]
 description-file = README.md
+[tool:pytest]
+python_classes =
```

Comment:
what does this do?... an empty "="?

Reply:
it makes it so that pytest fails to collect anything, so the warning gets surpressed
for whatever reason though, it still runs the tests, so I'm not sure what's up with that

Reply:
I think it's because:
```
Note that unittest.TestCase derived classes are always collected regardless of this option
```

Can we not use this option instead?
https://docs.pytest.org/en/latest/reference.html#confval-python_files

Reply:
I tried that, the issue with that is that we have mockbasetest classes, which cause issues here
I could move them out, but that seemed annoying to me >.>

Reply:
yeah, also, is this warning even worth addressing?...
i'm ok if we leave it...

Reply:
okay I did this a bit smarter, and it's only one class that I need to move, so that seems okayish to me